sample-apps/react/react-dogfood/components/ActiveCallHeader.tsx

```diff
@@ -38,26 +38,24 @@ const LatencyIndicator = () => {
   );
 };
 
-const Elapsed = ({ startedAt }: { startedAt: string | undefined }) => {
+const formatTime = (timeInSeconds: number) => {
+  const date = new Date(0);
+  date.setSeconds(timeInSeconds);
+  const format = date.toISOString(); // '1970-01-01T00:00:35.000Z'
+  const hours = format.substring(11, 13);
+  const minutes = format.substring(14, 16);
+  const seconds = format.substring(17, 19);
+  return `${hours !== '00' ? hours + ':' : ''}${minutes}:${seconds}`;
+};
+
+const Elapsed = () => {
   const [elapsed, setElapsed] = useState<string>();
-  const startedAtDate = useMemo(
-    () => (startedAt ? new Date(startedAt).getTime() : Date.now()),
-    [startedAt],
-  );
+  const { useCallDuration } = useCallStateHooks();
+  const duration = useCallDuration();
+
   useEffect(() => {
-    const interval = setInterval(() => {
-      const elapsedSeconds = (Date.now() - startedAtDate) / 1000;
-      const date = new Date(0);
-      date.setSeconds(elapsedSeconds);
-      const format = date.toISOString(); // '1970-01-01T00:00:35.000Z'
-      const hours = format.substring(11, 13);
-      const minutes = format.substring(14, 16);
-      const seconds = format.substring(17, 19);
-      const time = `${hours !== '00' ? hours + ':' : ''}${minutes}:${seconds}`;
-      setElapsed(time);
-    }, 1000);
-    return () => clearInterval(interval);
-  }, [startedAtDate]);
+    setElapsed(formatTime(duration));
+  }, [duration]);
```

Comment:
I think we can remove the `useEffect` and `useState` here, and apply the formatting directly:

```ts
const elapsed = formatTime(duration);
```